client/index.html

```diff
@@ -198,16 +198,15 @@ <h1 class="title">
 						</div>
 					</form>
 				</div>
-				<div id="settings" class="window">
+				<div id="settings" class="window" data-type="settings">
 					<div class="header">
 						<button class="lt" aria-label="Toggle channel list"></button>
 					</div>
 					<div class="container">
+						<h1 class="title">Settings</h1>
+
 						<div class="row">
 							<div class="col-sm-12">
-								<h1 class="title">Settings</h1>
-							</div>
-							<div class="col-sm-12">
 								<h2>Messages</h2>
```

Comment:
Do you want to take this title out of the `row`+`col-md-12` as well?

Reply:
All the headings are in inside. This isn't related to the PR either way.